# @generated
load("@antlir//antlir/antlir2/package_managers/dnf/rules:rpm.bzl", "rpm")
load("@antlir//antlir/antlir2/package_managers/dnf/rules:xml.bzl", "xml")
rpm(name='0-2021.1-5.el9.noarch-c77bf', rpm_name='python3-pytz', epoch=0, version='2021.1', release='5.el9', arch='noarch', url='https://mirror.facebook.net/centos-composes/9/CentOS-Stream-9-20250906.0/compose/AppStream/x86_64/os/Packages/python3-pytz-2021.1-5.el9.noarch.rpm', xml=':0-2021.1-5.el9.noarch-c77bf--xml', sha1=None, sha256='c77bf5578a2bcf7acaed8374ecfd452fe96bc29aa1ff1c47300f7560d70277ce', visibility=['//snapshot/rpm/centos9-compose/...'])
xml(name='0-2021.1-5.el9.noarch-c77bf--xml', primary='<package type="rpm">\n  <name>python3-pytz</name>\n  <arch>noarch</arch>\n  <version epoch="0" ver="2021.1" rel="5.el9"/>\n  <checksum type="sha256" pkgid="YES">c77bf5578a2bcf7acaed8374ecfd452fe96bc29aa1ff1c47300f7560d70277ce</checksum>\n  <summary>World Timezone Definitions for Python</summary>\n  <description>pytz brings the Olson tz database into Python. This library allows accurate\nand cross platform timezone calculations using Python 2.3 or higher. It\nalso solves the issue of ambiguous times at the end of daylight savings,\nwhich you can read more about in the Python Library Reference\n(datetime.tzinfo).\n\nAlmost all (over 540) of the Olson timezones are supported.</description>\n  <packager>builder@centos.org</packager>\n  <url>http://pytz.sourceforge.net/</url>\n  <time file="1689708696" build="1689250584"/>\n  <size package="52167" installed="180371" archive="185764"/>\n  <location href="Packages/c77bf5578a2bcf7acaed8374ecfd452fe96bc29aa1ff1c47300f7560d70277ce/python3-pytz-0:2021.1-5.el9.noarch.rpm"/>\n  <format>\n    <rpm:license>MIT</rpm:license>\n    <rpm:vendor>CentOS</rpm:vendor>\n    <rpm:group>Unspecified</rpm:group>\n    <rpm:buildhost>x86-04.stream.rdu2.redhat.com</rpm:buildhost>\n    <rpm:sourcerpm>pytz-2021.1-5.el9.src.rpm</rpm:sourcerpm>\n    <rpm:header-range start="8280" end="15685"/>\n    <rpm:provides>\n      <rpm:entry name="python-pytz" flags="EQ" epoch="0" ver="2021.1" rel="5.el9"/>\n      <rpm:entry name="python3-pytz" flags="EQ" epoch="0" ver="2021.1" rel="5.el9"/>\n      <rpm:entry name="python3.9-pytz" flags="EQ" epoch="0" ver="2021.1" rel="5.el9"/>\n      <rpm:entry name="python3.9dist(pytz)" flags="EQ" epoch="0" ver="2021.1"/>\n      <rpm:entry name="python3dist(pytz)" flags="EQ" epoch="0" ver="2021.1"/>\n    </rpm:provides>\n    <rpm:requires>\n      <rpm:entry name="python(abi)" flags="EQ" epoch="0" ver="3.9"/>\n      <rpm:entry name="tzdata"/>\n    </rpm:requires>\n    <rpm:obsoletes>\n      <rpm:entry name="python-pytz" flags="LT" epoch="0" ver="2021.1" rel="5.el9"/>\n      <rpm:entry name="python39-pytz" flags="LT" epoch="0" ver="2021.1" rel="5.el9"/>\n    </rpm:obsoletes>\n  </format>\n</package>\n', filelists='<package pkgid="c77bf5578a2bcf7acaed8374ecfd452fe96bc29aa1ff1c47300f7560d70277ce" name="python3-pytz" arch="noarch">\n  <version epoch="0" ver="2021.1" rel="5.el9"/>\n  <file type="dir">/usr/lib/python3.9/site-packages/pytz</file>\n  <file type="dir">/usr/lib/python3.9/site-packages/pytz-2021.1-py3.9.egg-info</file>\n  <file>/usr/lib/python3.9/site-packages/pytz-2021.1-py3.9.egg-info/PKG-INFO</file>\n  <file>/usr/lib/python3.9/site-packages/pytz-2021.1-py3.9.egg-info/SOURCES.txt</file>\n  <file>/usr/lib/python3.9/site-packages/pytz-2021.1-py3.9.egg-info/dependency_links.txt</file>\n  <file>/usr/lib/python3.9/site-packages/pytz-2021.1-py3.9.egg-info/top_level.txt</file>\n  <file>/usr/lib/python3.9/site-packages/pytz-2021.1-py3.9.egg-info/zip-safe</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/__init__.py</file>\n  <file type="dir">/usr/lib/python3.9/site-packages/pytz/__pycache__</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/__pycache__/__init__.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/__pycache__/__init__.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/__pycache__/exceptions.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/__pycache__/exceptions.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/__pycache__/lazy.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/__pycache__/lazy.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/__pycache__/reference.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/__pycache__/reference.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/__pycache__/tzfile.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/__pycache__/tzfile.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/__pycache__/tzinfo.cpython-39.opt-1.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/__pycache__/tzinfo.cpython-39.pyc</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/exceptions.py</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/lazy.py</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/reference.py</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/tzfile.py</file>\n  <file>/usr/lib/python3.9/site-packages/pytz/tzinfo.py</file>\n  <file type="dir">/usr/share/doc/python3-pytz</file>\n  <file>/usr/share/doc/python3-pytz/README.rst</file>\n  <file type="dir">/usr/share/licenses/python3-pytz</file>\n  <file>/usr/share/licenses/python3-pytz/LICENSE.txt</file>\n</package>\n', other='<package pkgid="c77bf5578a2bcf7acaed8374ecfd452fe96bc29aa1ff1c47300f7560d70277ce" name="python3-pytz" arch="noarch">\n  <version epoch="0" ver="2021.1" rel="5.el9"/>\n  <changelog author="Mohan Boddu &lt;mboddu@redhat.com&gt; - 2021.1-4" date="1628596800">- Rebuilt for IMA sigs, glibc 2.34, aarch64 flags\n  Related: rhbz#1991688</changelog>\n  <changelog author="Lum&#xED;r Balhar &lt;lbalhar@redhat.com&gt; - 2021.1-5" date="1687867200">- Fix FTBFS with newest tzdata\nResolves: RHEL-676</changelog>\n</package>\n')
